apache/rocketmq/v2/service.proto

```diff
@@ -183,9 +183,8 @@ message VerifyMessageResult {
 }
 
 message RecoverOrphanedTransactionCommand {
-  MessageQueue message_queue = 1;
-  Message orphaned_transactional_message = 2;
-  string transaction_id = 3;
+  Message orphaned_transactional_message = 1;
```

Comment:
Rename orphaned_transactional_message -> message as it's already wrapped in the structure of OrphanTransactionCommand